### DISCUSSION TITLE
Change the theme toggle button colour

### DISCUSSION THREAD

**Discussion by kahnuk** [adityatelange/hugo-PaperMod#965] [Q&A]
Hi,

I've spent some time digging around the theme's CSS and the repository discussions (#204 seems close but I can't for the life of me figure out where currentColor is defined & it's old) but haven't found where I would be able to change the colour of the theme button.

For context, I'd like to change the sun to a nice yellow :)

Any help would be much appreciated!

Thank you!

**Comment by adityatelange** [comment]
#204 seems unrelated 

`CurrentColor` is not a variable but css property which uses the color used in current context. You can google to know more about `CurrentColor`.

For changing color u can use the css class and add color to for ex. themetoggle.svg
